src/multi-level-store.ts

```diff
@@ -50,8 +54,8 @@ export function createMultiLevelStore<T extends Defined>(
 		...baseOpts,
 		...global,
 		store: redisStore,
-		host: REDIS.general.host,
-		port: REDIS.general.port,
+		host: redisOpts.host,
+		port: redisOpts.port,
```

Comment:
There's a bit of code a few lines above that Github won't let me comment on because it's not in the diff but I think should be changed now that you've removed the concept of "default", "logs", etc.

This:

```
	if (!('default' in opts)) {
		opts = { default: opts };
	}
	const { default: baseOpts, local, global } = opts;
```

should likely be changed to:

```
	const { local, global } = opts;
```